go.mod

```diff
@@ -29,5 +29,6 @@ require (
 	github.com/libp2p/go-libp2p-core v0.5.0
 	github.com/stretchr/testify v1.5.1
 	github.com/whyrusleeping/cbor-gen v0.0.0-20200810223238-211df3b9e24c
+	go.uber.org/atomic v1.6.0
```

Comment:
did you mean to add this? I don't have an objection. I haven't worked with this library before but I'm sure I can figure it out.

Reply:
Yeah, it's a simple wrapper over "sync/atomic.